<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableColumnEditor</class>
 <widget class="QWidget" name="TableColumnEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Column Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Column Data Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_type"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="check_isIndex">
     <property name="text">
      <string>Is the index column?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_isRef">
     <property name="text">
      <string>Is a reference column?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Referenced Table:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_refTable"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Referenced Column:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_refColumn"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_delete">
     <property name="text">
      <string>Delete Column</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
